assets/circleci.config.yml

```diff
@@ -16,6 +16,29 @@ parameters:
 
 workflows:
   version: 2
+  # This workflow run jobs to help improve on the quality of the codebase
+  # Will be run on issue and feature branches
+  # code-quality-assurance:
```

Comment:
@jfauske I'm wondering if we should enable this by default. This version of the YAML most likely only new projects will get by default and that's what we want, right?

Reply:
> @jfauske I'm wondering if we should enable this by default. This version of the YAML most likely only new projects will get by default and that's what we want, right?

I've reconsidered... Uncommented in a9d9880